packages/sdk/test/unit/document/document_size_test.ts

```
@@ -0,0 +1,105 @@
+
+  it('array test', function () {
+    const doc = new Document<{ arr: Array<string> }>('test-doc');
+
+    doc.update((root) => (root['arr'] = []));
+    assert.deepEqual(doc.getDocSize().live, { data: 0, meta: 72 });
+
+    doc.update((root) => root['arr'].push('a'));
+    assert.deepEqual(doc.getDocSize().live, { data: 2, meta: 96 });
+    assert.deepEqual(doc.getDocSize().gc, { data: 0, meta: 0 });
+
+    doc.update((root) => delete root['arr'][0]);
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)